source/common/stats/stats_impl.h

```diff
@@ -369,6 +373,126 @@ class HistogramImpl : public Histogram, public MetricImpl {
   Store& parent_;
 };
 
+/**
+ * Structure to hold statistical data of histogram.
+ */
+struct HistogramStatistics {
+public:
+  HistogramStatistics(histogram_t* histogram_ptr) {
+    double quantile_in[] = {0, 0.25, 0.5, 0.75, 0.90, 0.95, 0.99, 0.999, 1};
```

Comment:
factor out this constant so it only appears once.  I see it also in grpc_metrics_service_impl.h:149

Reply:
i am not able to make it const - because of my c++ knowledge levels :-). But made it class variable and used it every where.